### ISSUE TITLE
Initialization SQLConnection with custom host only from Properties.

### ISSUE THREAD

**Issue by Gennady163sam** [tarantool/tarantool-java#234]
When we create _new NamedParameterJdbcTemplate(new DriverManagerDataSource("jdbc:tarantool://#some-url#:3301"))_ with some url (not localhost), we get the following error in a console

```
Caused by: java.sql.SQLException: Couldn't connect to localhost:3301
	at org.tarantool.jdbc.SQLConnection.getConnectedSocket(SQLConnection.java:116)
	...

```
Class _org.tarantool.jdbc.SQLConnection_ which get custom url in constructor, doesn't used it.
Method "_getConnectedSocket()_" from class _org.tarantool.jdbc.SQLConnection_ using host and port only from Properties.

```
111         String host = properties.getProperty(PROP_HOST);
112         int port = Integer.parseInt(properties.getProperty(PROP_PORT));
```

Steps:
```
NamedParameterJdbcTemplate template = new NamedParameterJdbcTemplate(new DriverManagerDataSource("jdbc:tarantool://#some-url#:3301"));
RowMapper<Object> rowMapper = new RowMapper<Object>() {
    @Override
    public Object mapRow(ResultSet resultSet, int i) throws SQLException {
        return Arrays.asList(resultSet.getInt(1), resultSet.getString(2));
    }
};
System.out.println(template.query("select * from hello_world", rowMapper));
```

**Comment by nicktorwald** [comment]
Thanks for your vigilance.
What the connector version did you use?
I guess it was the latest released `1.9.2` or below because I could reproduce this behavior only on those versions.
If so, you have three options to overcome this issue:
1. Use `tarantool:\\` URL scheme instead of `jdbc:tarantool:\\` (yes, this solution violates the JDBC standard). See an example [for 1.9.2](https://github.com/tarantool/tarantool-java/tree/903cb429eb8c688be7807a3fd3df0ca665f2e214#spring-namedparameterjdbctemplate-usage-example).
2. Use `Properties` object to configure the right address using `new DriverManagerDataSource(connectionString, props)`.
3. Use the current snapshot version where `jdbc:tarantool:\\` is handled correctly by the driver.

Hope, it helps you.

**Comment by Totktonada** [comment]
So the question is about support of `jdbc:tarantool://` schema? It was fixed in 2c9cd80dacb4a40a8818295eca00c1370bdf628a?

**Comment by nicktorwald** [comment]
> So the question is about support of `jdbc:tarantool://` schema? It was fixed in [2c9cd80](https://github.com/tarantool/tarantool-java/commit/2c9cd80dacb4a40a8818295eca00c1370bdf628a)?

Exactly, prior to [2c9cd80](https://github.com/tarantool/tarantool-java/commit/2c9cd80dacb4a40a8818295eca00c1370bdf628a) `jdbc:tarantool:\\` was unsupported schema.

**Comment by Gennady163sam** [comment]
I did use 1.9.2 version.
Thanks for reply. Your options work for me.

**Comment by Totktonada** [comment]
We just released [connector-1.9.3](https://github.com/tarantool/tarantool-java/releases/tag/connector-1.9.3), where the problem should gone. So, I'm closing the issue.
